webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/detail/AnnotationFeatureForm.java

```diff
@@ -818,4 +843,29 @@ protected void onUpdate(AjaxRequestTarget aTarget)
             };
         }
     }
+
+    void setSelectedTag(String selectedTag)
```

Comment:
Hm, I personally don't use the package-private scope. Only either public, protected, or private.